crates/pallet-executor/src/lib.rs

```diff
@@ -231,6 +231,16 @@ mod pallet {
 
             Ok(())
         }
+
+        #[pallet::weight(10_000)]
```

Comment:
As I commented in another place, we may not need this if we just have a constant and can do runtime upgrade. Also random weights should always have TODOs on them.

Reply:
A runtime upgrade is definitely less convenient than a sudo call if we consider this parameter is not pretty stable and could be tweaked more likely.